docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
Sure let's work through it.

What happens when you run `gulp website:serve`? Is there an error you could show me?